mocharc.coverage.js

```diff
@@ -0,0 +1,11 @@
+module.exports = Object.assign(
+  {},
+  require('./mocharc.js'),
+  {
+    //recursive: true, // fails
```

Comment:
why does this fail?

Reply:
seems with `recursive` the entry file `test/test.js` is treated as glob pattern
and all files in `test/` are called **recursive**ly

its just a relic from the sample config

Reply:
ok. I think it would be best to just remove it in that case